src/foam/util/geo/GeolocationSupport.js

```diff
@@ -36,7 +38,14 @@ foam.CLASS({
     private final static GeolocationSupport instance__ = new GeolocationSupport();
     public static GeolocationSupport instance() {
       var ret = instance__;
-      init(foam.core.XLocator.get(), ret);
+      X x = foam.core.XLocator.get();
+      X resourceStorageX = x;
```

Comment:
@lchanmann can you instead modify foam/nanos/boot/Boot.java and create the appropriate Storage class there, thanks.